src/media/Season.java

```diff
@@ -0,0 +1,7 @@
+package media;
+
+import java.util.TreeSet;
+
+public class Season extends Series {
+    private TreeSet<Episode> episodes = new TreeSet<>();
```

Comment:
Brug `Set` interface i stedet for `TreeSet` klassen
```suggestion
    private Set<Episode> episodes = new TreeSet<>();
```